### PR TITLE
update ruble sign

### PR DESCRIPTION
**Pull request by sanekyy**
Hello. I have update ruble sign to official. 

About ruble sign [here](https://en.wikipedia.org/wiki/Ruble_sign).

New sign: ₽ ( U+20BD )

Related issue: #262

### CODE REVIEW

**Comment by marinat** [comment]
Please, repeat review of pr.

**Comment by kevmoo** [comment]
@sanekyy – please fix tests!

Also: this should change should be mentioned in the changelog.

CC @davidmorgan

**Comment by sanekyy** [comment]
@kevmoo fixed.

**Comment by sanekyy** [comment]
@kevmoo friendly ping

any news?

**Comment by srawlins** [comment]
The owners file for the intl package internally lists two Googlers.

**Comment by otopba** [comment]
Any updates? =/

1 year has already passed

**Comment by sanekyy** [comment]
Hello @kevmoo @davidmorgan @srawlins 

Can you help to merge this PR?

one and half year waiting:(

**Comment by mosuem** [comment]
This needs merge conflict resolution, but LGTM apart from that, as we don't get the currency symbols from CLDR.

File: test/number_format_compact_test.dart [hunk 1]
```diff
@@ -8,6 +8,9 @@ import 'package:intl/intl.dart';
 import 'package:intl/number_symbols_data.dart' as patterns;
 import 'package:test/test.dart';
 
+import 'package:test/test.dart';
```
Comment (mosuem) [inline test/number_format_compact_test.dart:11]: Why re-import the package?

File: AUTHORS [hunk 2]
```diff
@@ -7,3 +7,4 @@
 #   Organization <fnmatch pattern>
 #
 Google Inc. <*@google.com>
+Aleksandr Yurkovskiy sanekyy@gmail.com
```
Comment (mosuem) [inline AUTHORS:10]: Do we actually use the `AUTHORS` file like this @devoncarew ?

**Review by mosuem — CHANGES_REQUESTED** [review]

**Review by mosuem — APPROVED** [review]

**Pull request merged**